### ISSUE TITLE
Error when rescanning local assets if asset folder is removed

### ISSUE THREAD

**Issue by filigran** [Wintermute0110/plugin.program.AEL#76]
Didn't see an issue about this, apologies if it's a duplicate:

Migrating from AL to AEL, so not really used to this asset stuff. Noticed something when rescanning assets for a rom launcher. I removed all unnecessary asset folders that were autocreated, leaving just boxfronts and fanart (the ones I use in my skin), and AEL crashed when rescanning:
```
22:19:24.244 T:1440   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
- NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
Error Type: <type 'exceptions.WindowsError'>
Error Contents: [Error 3] Det g? inte att hitta s?kv?en: u'D:\\Thomas\\Portabla program\\Kodi17.6_Krypton\\portable_data\\emulators\\roms\\nes\\assets\\titles\\*.*'
Traceback (most recent call last):
    File "D:\Thomas\Portabla program\Kodi17.6_Krypton\portable_data\addons\plugin.program.advanced.emulator.launcher\addon.py", line 39, in <module>
       main.Main().run_plugin()
    File "D:\Thomas\Portabla program\Kodi17.6_Krypton\portable_data\addons\plugin.program.advanced.emulator.launcher\resources\main.py", line 270, in run_plugin
       self.run_protected(command, args)
    File "D:\Thomas\Portabla program\Kodi17.6_Krypton\portable_data\addons\plugin.program.advanced.emulator.launcher\resources\main.py", line 358, in run_protected
       self._command_edit_launcher(args['catID'][0], args['launID'][0])
    File "D:\Thomas\Portabla program\Kodi17.6_Krypton\portable_data\addons\plugin.program.advanced.emulator.launcher\resources\main.py", line 1674, in _command_edit_launcher
       misc_add_file_cache(launcher[AInfo.path_key])
    File "D:\Thomas\Portabla program\Kodi17.6_Krypton\portable_data\addons\plugin.program.advanced.emulator.launcher\resources\utils.py", line 386, in misc_add_file_cache
       file_list = os.listdir(dir_FN.getPath())
WindowsError: [Error 3] Det g? inte att hitta s?kv?en: u'D:\\Thomas\\Portabla program\\Kodi17.6_Krypton\\portable_data\\emulators\\roms\\nes\\assets\\titles\\*.*'
-->End of Python script error report<--
```
The error "Det g? inte att hitta s?kv?en" translates to "Cannot find file path", meaning: I removed ...\assets\titles and now it errors out. Needs some more error handling to cover that I suppose.

**Comment by Wintermute0110** [comment]
Thanks for reporting.

The migration from AL to AEL is not supported any more. I haven't used AL for the last 18 months at least. Also, AL and AEL have diverged so much, the migration is difficult specially because the assets. I recommend that you rebuild your setup and create dedicated asset directories for AEL. You can then copy your fanarts, boxfronts, etc. from AL to AEL assets directories.

In any case, the issue you report is a bug. I will fix it ASAP.

**Comment by Wintermute0110** [comment]
I think this issue is solved in 0.9.8-beta1. Reopen if not.
